<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>203</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>254</width>
    <height>203</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>254</width>
    <height>203</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lanchonete</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-image: url(:/imagens/red.jpg);
}

QGroupBox
{
	font: bold 8pt &quot;Noto Sans&quot;;
	border: 2px solid black;
}

QRadioButton:checked
{
	color: green;
}

QCheckBox:checked
{
	color: blue;
}</string>
  </property>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>151</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAjuda">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajuda</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splSeparador">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>235</width>
     <height>129</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QGroupBox" name="gpbLanches">
    <property name="title">
     <string>Lanches</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="rbtSalada">
       <property name="text">
        <string>xSalada</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbtFrango">
       <property name="text">
        <string>xFrango</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbtTudo">
       <property name="text">
        <string>xTudo</string>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>rbtSalada</zorder>
    <zorder>rbtTudo</zorder>
    <zorder>rbtFrango</zorder>
    <zorder>rbtSalada</zorder>
   </widget>
   <widget class="QGroupBox" name="gpbBebida">
    <property name="title">
     <string>Bebidas</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="chkSemGelo">
       <property name="text">
        <string>Sem Gelo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkLight">
       <property name="text">
        <string>Light</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkDiet">
       <property name="text">
        <string>Diet</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <zorder>gpbLanches</zorder>
  <zorder>gpbBebida</zorder>
  <zorder>btnCancelar</zorder>
  <zorder>btnOK</zorder>
  <zorder>btnOK</zorder>
  <zorder>btnOK</zorder>
  <zorder>btnAjuda</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
